src/renderer/base/FontInfoDesired.cpp

```diff
@@ -8,23 +8,24 @@
 FontInfoDesired::FontInfoDesired(const std::wstring_view& faceName,
                                  const unsigned char family,
                                  const unsigned int weight,
-                                 const til::size coordSizeDesired,
+                                 const float fontSize,

```

Comment:
we used width + height for a specific conhost reason. will this break that?

Reply:
The width/height combination is for raster fonts which we don't use in Windows Terminal. conhost doesn't use this constructor.